mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1632,6 +1675,93 @@ trait Completions { this: MetalsGlobal =>
         }
       }
     }
+
+    /**
+     * A scaladoc completion showing the parameters of the given associated definition.
+     *
+     * @param editRange the range in the original source file.
+     * @param associatedDef the memberDef associated with the scaladoc to complete.
+     *                      This class will construct scaladoc based on the params of this definition.
+     */
+    case class Scaladoc(
+        editRange: l.Range,
+        associatedDef: MemberDef
+    ) extends CompletionPosition {
+      override def contribute: List[Member] = {
+        val lines = scaladocLines(associatedDef)
+        // add `* @return` line only if the definition is a method def.
+        val returnLine =
+          if (lines.isEmpty && clientSupportsSnippets) "* @return $0"
```

Comment:
I meant to move the cursor to `@return | <- here` only if there are no parameters.
But now I'm going to move the cursor to the following position, we don't need this logic anymore.

```scala
/**
   * | <- here instead of after the @param
   *
   * @param
   */
```